src/sentry/web/frontend/group_event_json.py

```diff
@@ -1,17 +1,47 @@
 from django.http import Http404, HttpRequest, HttpResponse
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import OpenApiParameter, extend_schema
 
 from sentry import eventstore
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.apidocs.constants import RESPONSE_NOT_FOUND
 from sentry.models.group import Group, get_group_with_redirect
 from sentry.models.groupmeta import GroupMeta
 from sentry.utils import json
 from sentry.web.frontend.base import OrganizationView, region_silo_view
 
 
+@extend_schema(tags=["Events"])
 @region_silo_view
 class GroupEventJsonView(OrganizationView):
+    publish_status = {
+        "GET": ApiPublishStatus.PUBLIC,
+    }
+    owner = ApiOwner.ISSUES
     required_scope = "event:read"
 
+    @extend_schema(
+        operation_id="Retrieve the JSON object for an Event",
+        description="Returns the JSON object for an event.",
+        parameters=[
+            OpenApiParameter(
```

Comment:
could we move the parameter definition to https://github.com/getsentry/sentry/blob/master/src/sentry/apidocs/parameters.py

i see there are a couple other endpoints who don't do this, i can make a pr to move them over.

Reply:
Done